angularjs-portal-frame/src/main/webapp/portal/misc/directives.js

```diff
@@ -8,6 +8,7 @@ define(['angular', 'require'], function(angular, require) {
      * REQUIRED attribute that isn't listed below:
```

Comment:
I don't get what "attribute that isn't listed below" means, now that I read this.  (Acknowledging that that wasn't added by this changeset. :smile:)